### ISSUE TITLE
[SDK46] - dev client - expo run:ios not launching iOS app inside simulator

### ISSUE THREAD

**Issue by sync** [expo/expo#18536]
## expected

app to be installed and then launched into current iOS simulator

## actual

app is installed but never launched...

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

iOS

### SDK Version (managed workflow only)

46

### Environment

```
expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 12.5
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.16.0 - ~/Library/Caches/fnm_multishells/61970_1659690967341/bin/node
      Yarn: 1.22.19 - ~/Library/Caches/fnm_multishells/61970_1659690967341/bin/yarn
      npm: 8.15.1 - ~/Library/Caches/fnm_multishells/61970_1659690967341/bin/npm
      Watchman: 2022.07.04.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.11.2 - /Users/anthonymittaz/.gem/ruby/2.7.4/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: 2021.2 AI-212.5712.43.2112.8609683
      Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
    npmGlobalPackages:
      eas-cli: 0.57.0
      expo-cli: 6.0.1
    Expo Workflow: managed
```

### Reproducible demo

create a new project with: `npx create-expo-app --template expo-template-blank-typescript@sdk-46`

add dev client: `yarn expo install expo-dev-client`

then run: `expo run:ios`

## expected
app to be installed and then launched into current iOS simulator

## actual 
app is installed but never launched... 

```
› Metro waiting on
com.sync.pluviae://expo-development-client/?url=http%3A%2F%2F192.168.4.52%3A8081
› Scan the QR code above with Expo Go (Android) or the Camera app (iOS)

› Press a │ open Android
› Press i │ open iOS simulator
› Press w │ open web

› Press r │ reload app
› Press m │ toggle menu

› Press ? │ show all commands

› Installing on iPhone 13 mini
```

If I follow those exact same steps using the SDK 45, the app get installed and then launched inside my current iOS simulator.

Reproduced here: https://github.com/sync/-fake-app-expo-sdk46-issue
[fake-app.zip](https://github.com/expo/expo/files/9274351/fake-app.zip)

**Comment by sync** [comment]
I was debuting the cli and saw this: 

https://github.com/expo/expo/blob/6f9afd41bac6a41c29f7afa2468db9c70d9de953/packages/%40expo/cli/src/start/platforms/ios/AppleDeviceManager.ts#L144

```
private async getApplicationIdFromBundle(filePath: string): Promise<string> {
    // TODO: Implement...
    return EXPO_GO_BUNDLE_IDENTIFIER;
  }
```

And it seems that this is what's happening could it be looking for the wrong appId ? 

```
waiting for simulator to install
{ appId: 'host.exp.Exponent' }
```

**Comment by sync** [comment]
Here is the path-package fix if anyone is experiencing the same issue: 
[@expo+cli+0.2.6.patch.zip](https://github.com/expo/expo/files/9274684/%40expo%2Bcli%2B0.2.6.patch.zip)

**Comment by sync** [comment]
See PR here: https://github.com/expo/expo/pull/18537/files

**Comment by lukmccall** [comment]
I believe that fix was already published with the `expo@46.0.7`.
